src/lib/gundb/UserStorageClass.js

```diff
@@ -736,7 +736,15 @@ export class UserStorage {
       keys(profileSettings)
         .filter(key => profile[key])
         .map(async field => {
-          return this.setProfileField(field, profile[field], await getPrivacy(field)).catch(e => {
+          let privacy
+
+          if (Config.isEToro && !update) {
```

Comment:
no need for this. just change the defaultprivacy in profileSettings above

Reply:
Done